### ISSUE TITLE
windows 64 bit version 1.6.0 winvnc4.exe -help does not work

### ISSUE THREAD

**Issue by oldunixguy** [TigerVNC/tigervnc#319]
Clearly the src code shows -help command line option. When I issue winvnc4.exe -help on windows 7 pro nothing happens.

regards,
oldunixguy

**Comment by CendioOssman** [comment]
Yeah, this is an old issue. The code is there but I don't think it has ever worked because of how the console works in Windows. It is functional in the debug builds, but that's in most cases not helpful.

**Comment by oldunixguy** [comment]
thanks.
I'm struggling with getting this server working on win7. no matter what I
try I continue to get authentication errors when trying to get in. So I'm
poking around the code looking for help. I have scoured the web and find
nothing that works.
regards
oldunixguy

On Mon, May 30, 2016 at 6:27 AM, Pierre Ossman (Work account) <
notifications@github.com> wrote:

> Yeah, this is an old issue. The code is there but I don't think it has
> ever worked because of how the console works in Windows. It is functional
> in the debug builds, but that's in most cases not helpful.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/TigerVNC/tigervnc/issues/319#issuecomment-222481802>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AMMAj_sWLrROBGL_ZzJf0ATjxM2jXHCLks5qGteagaJpZM4IoJ2a>
> .
>



-- 
Richard A. Painter, P.E.

Homeless from the Black Forest Colorado Forest Fire

AMERICA- Designed by geniuses, Run by idiots, fools and criminals!

**Comment by CendioOssman** [comment]
You can enable logging using either `-Log *:file:100` to get a file in `c:\temp`, or `-Log *:EventLog:100` to get it in the Windows event log.
